.travis.yml

```diff
@@ -2,61 +2,75 @@ sudo: false
 
 language: php
 
-branches:
-  except:
-    - /^release-\d+\.\d+\.\d+.*$/
-    - /^ghgfk-.*$/
-
 cache:
   directories:
     - $HOME/.composer/cache
-    - $HOME/.local
-    - zf-mkdoc-theme
 
 env:
   global:
-    - SITE_URL: https://zendframework.github.io/zend-soap
-    - GH_USER_NAME: "Matthew Weier O'Phinney"
-    - GH_USER_EMAIL: matthew@weierophinney.net
-    - GH_REF: github.com/zendframework/zend-soap.git
-    - secure: "O2ECNTc7cif3sN3Iyjiheu4UsHcs4g/TrNtc5XqcTpecxcM4ncG1oLAoF62ZN2cwdpJ8nBwl3BQDI9nkLS+nxPHxdqV5bobZ/8ZcI1/2by+l9N720WQfezPxxSxAAPl5b1SXxocsWUiykJJpADqGdYjgfmgAyiK70tXN4XwZl27Svnh+ulqFB2aYQNUv28jJZ3/Pt8jW5sgTVYKYZ65bYirEHGMsUnOK5nJ/W04qoq1d8Qj5x4xTrA71KecibljXpqXHtrQRBpGoKmU8N0w7XdGn0l6xojEJ3QkLE1/1SrZq0r4GHgP294358IsepdQXoqAIxULR0nlNkLmDY//wQI6WRWOpTI+eYN+unoq/zDgPnZGzV6NX/BrRt1yxhNij8CpIvR8cg4WP52Cl2djG5YWr5sOOqffIvSDKJLZgUOb512F7DdI2Pr6HSDyOSqrKoMGMpLJb7/1sEumLfVGmpu9fgpWakxR5wTa0oMBZknyJVy18nj+mj5YgIzD+1FYzYj5ttajh3KKaxDgTCYHJkCA/MRONsr/Ap1oHI2W5iDgAEyaJhru0XefNjW37n/sx3QqZ+F4VZDbuTy8FaFMbnT6+brbCLdrxtmIwo2OhYob+G5rWQejnF81g1ShUE/H3l/rlQ+BLZzwYEgAHkJ5YTAQYq37E3Ra7DgOv14Ayjvo="
+    - COMPOSER_ARGS="--no-interaction"
+    - COVERAGE_DEPS="satooshi/php-coveralls"
```

Comment:
when the older prs are okay, than i change all other prs. atm its bit strange to create a pr and 2 or 3 or 4 months later something changed and i have to update all once. this take a bit time.


also that lock file is generate with php5.6 so its not needed, have to change everything but i want a confirmation of the old prs like https://github.com/zendframework/zend-stdlib/pull/74